### PR TITLE
Remove redundant minOccurs

### PR DESCRIPTION
**Pull request by ykulbak**

### CODE REVIEW

**Review by pasi-paasiala — APPROVED** [review]
+1

File: Schemas/markup.xsd [hunk 1]
```diff
@@ -11,6 +11,13 @@
...
     <xs:complexType name="Header">
         <xs:sequence>
+            <xs:element name="Files">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="File" type="File" minOccurs="0" maxOccurs="unbounded"/>
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
             <xs:element name="File" maxOccurs="unbounded">
```
Comment (jasollien) [inline Schemas/markup.xsd:21]: Must also remove this

File: Schemas/markup.xsd [hunk 2]
```diff
@@ -58,7 +65,8 @@
             <xs:element name="ReferenceLinks" minOccurs="0">
                 <xs:complexType>
                     <xs:sequence>
-                        <xs:element name="ReferenceLink" type="NonEmptyOrBlankString" minOccurs="0" maxOccurs="unbounded"/>
+                        <xs:element name="ReferenceLink" type="NonEmptyOrBlankString" minOccurs="0"
+                                    maxOccurs="unbounded"/>
```
Comment (jasollien) [inline Schemas/markup.xsd:69]: Why this line break?

**Review by jasollien — CHANGES_REQUESTED** [review]

**Comment by pasi-paasiala** [comment]
This PR has extra commits. It should be rebased.

**Review by pasi-paasiala — APPROVED** [review]

**Pull request merged**